### ISSUE TITLE
[FSR] Incorporate MCP debts into FSR (IXD)

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#29980]
## Issue Description
There are a few places in the FSR that need interaction updates in order to incorporate copays into the form: 
- Debt selection
- Resolution selection

---
## Tasks
- [x] Create designs to incorporate copay debts and sync to UXPin
- [x] Share work to team 
- [ ] ~Create FE tickets (if needed)~

## Acceptance Criteria
- [x] Design created and UXPin linked to this ticket
- [x] Shared design to team
- [ ] ~FE tickets created and linked to this ticket (if needed)~

**Comment by rebeccawalsh** [comment]
This work has been broken into two tickets to account for some discussions that need to happen prior to SME meetings to confirm accuracy. This ticket covers just interaction design changes to the debt selection page and the resolution selection pages.

**Comment by rebeccawalsh** [comment]
Work for this ticket can be seen in all chapter 5 entries in UXPin [here](https://preview.uxpin.com/d8307359c0a22d9e09b86d4803be3dce030d24ff#/pages/142617671/simulate/sitemap?mode=i)!

This work was demo'd October 5, and a message in our team chat for all team members to review!

**Comment by denisecoveyduc** [comment]
Closing this out as future work will be captured in #30698
